src/modules/job-manager/event.c

```diff
@@ -675,6 +675,8 @@ int event_job_post_pack (struct event *event,
         return -1;
     if (event_batch_process_event_entry (event, job, name, entry) < 0)
         goto error;
+    if (EVENT_INFO_ONLY & flags)
+        goto out;
```

Comment:
Suggestion: could we call it "journal" instead of "event stream" in the commit message, and maybe call the flag EVENT_JOURNAL_ONLY?   Consistent naming can make things clearer.